medical-portal/src/API/RSBC.DMF.MedicalPortal.API/Controllers/CasesController.cs

```diff
@@ -137,6 +138,18 @@ private string TranslateDmerStatus(string dmerStatus, string loginId)
                     dmerStatus = "Required - Claimed";
                 }
             }
+
+            if(dmerStatus == "Non-Comply")
+            {
+                if (string.IsNullOrEmpty(loginId))
+                {
+                    dmerStatus = "Non-Comply - UnClaimed";
```

Comment:
typo, the word is spelled "unclaimed", I had to google it to verify. There is another instance below

Reply:
Fixed The typo in both the places